.github/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Changes from release 2022/13 to 2023/01
+1. Enchancement - Changes made to Liverpool (EGGP) SMR holding point and taxiway labels changed from red to white - thanks to @zippy77777 (Darren Faux)
```

Comment:
Sorry to be picky, but to make this consistent with the style guide, changelog entries should be in the past tense and consise (sorry I didn't spot this when guiding you through it, screen sharing was being fun).
```suggestion
1. Enchancement - Changed Liverpool (EGGP) SMR holding point labels from red to white - thanks to @zippy77777 (Darren Faux)
```